conn.c

```diff
@@ -33,13 +33,13 @@ make_conn(int fd, char start_state, tube use, tube watch)
     job j;
     Conn *c;
 
-    c = new(Conn);
-    if (!c) return twarn("OOM"), NULL;
+    c = (Conn *)new(Conn);
```

Comment:
Out of curiosity, what happens if you run

```
cc -DNULL='((void*)0)' -Wall -Werror -Wformat=2 -g    -c -o conn.o conn.c
```

on the original file without these changes?